docs/tutorials_examples/index.rst

```diff
@@ -1,13 +1,17 @@
-﻿Tutorials
-=========
+﻿Tutorials and Examples
+======================
 
 
-PARETO tutorials are currently under development and will be made publicly available as soon as possible.
+PARETO currently has the following Jupyter notebook demonstrations available:
 
-Since PARETO project is an open source project and we are in collaboration with WaterTAP and IDAES projects, the IDAES tutorials and WaterTAP tutorials are good learning materials.
+1. `Strategic model demo <https://github.com/project-pareto/project-pareto/blob/main/pareto/examples/strategic_model_general_demo/strategic_model_general_demo.ipynb>`_
+2. `Strategic model - treatment module and sensitivity analysis demo <https://github.com/project-pareto/project-pareto/blob/main/pareto/examples/strategic_model_treatment_module_demo/strategic_model_treatment_module_demo.ipynb>`_
 
-In particular, new users should be familiar with Python and Pyomo. Because PARETO and IDAES share this common base, we refer users unfamiliar with Python or Pyomo to the IDAES tutorials `Short Introduction to Python and Pyomo <https://idaes.github.io/examples-pse/latest/Tutorials/Basics/introduction_short_solution_doc.html>`_ or the `Longer Introduction to Python and Pyomo <https://idaes.github.io/examples-pse/latest/Tutorials/Basics/introduction_solution_doc.html>`_.
+We encourange new PARETO users to download and run these Jupyter notebooks. Additional Jupyter notebook tutorials and examples are currently under development and will be added to the `examples directory <https://github.com/project-pareto/project-pareto/tree/main/pareto/examples>`_ of the PARETO repository when they are available. The above list will be updated as well.
```

Comment:
This could be more of a general team discussion, but I'd be more specific here in describing the target audience, since "users" also applies to those who would only use PARETO through the GUI:

```suggestion
Users interested in utilizing PARETO programmatically by writing Python code can download and run these Jupyter notebooks. Additional Jupyter notebook tutorials and examples are currently under development and will be added to the `examples directory <https://github.com/project-pareto/project-pareto/tree/main/pareto/examples>`_ of the PARETO repository when they are available. The above list will be updated as well.
```

Reply:
@lbianchi-lbl I like the suggestion - I committed it. Thanks!